libfdt/libfdt.h

```diff
@@ -547,12 +548,17 @@ int fdt_path_offset_namelen(const void *fdt, const char *path, int namelen);
  *
  *   /soc@0/i2c@30a40000/eeprom@52
  *   i2c2/eeprom@52
+ *   &foo/eeprom@52
+ *   &bar
```

Comment:
... so (as `/soc@0` is the only `soc@????` in `/` and `/soc@0/i2c@30a40000` is the only `i2c@???` in `/soc@0`, and `eeprom@52` is also unique,

    /soc/i2c/eeprom

is another path to that node.

I'm unsure if this is a bug of a feature though.

Reply:
Well, in that very minimal example, yes, and presumably soc@0 is always the only soc node, but in real life, there'd very likely be several `i2c@...` nodes, and I don't see the point of confusing this example with the subtleties from that ambiguity, which has nothing to do with the feature I'm implementing.